### PR TITLE
fix building packages on travis-docker

### PR DESCRIPTION
**Pull request by ebbit1q**
Fix building packages on travis-docker #3433.

### CODE REVIEW

**Comment by ebbit1q** [comment]
Fixed, please review.

File: .ci/UbuntuBionic/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
```
Comment (ctrlaltca) [inline .ci/UbuntuBionic/Dockerfile]: Looks like you moved the Dockerfile to a subdirectory, but the old file in `.ci/` has not been deleted.

Comment (ebbit1q) [reply .ci/UbuntuBionic/Dockerfile]: strange, I guess `git mv` doesn't actually remove the old file?

**Comment by ctrlaltca** [comment]
File fixed, ci succeded

**Pull request merged**